src/utils/display.utils.js

```diff
@@ -0,0 +1,3 @@
+export const elipsify = (str, length) => {
```

Comment:
this can be done with css  `line-clamp` I think using css we can use up all the space and not worry how many characters fit in the card (because the card changes it's width on different resolutions). Here is an article explaining how to use it https://dev.to/temmietope/how-to-truncate-multi-line-string-with-pure-css-4fc9

Reply:
please check